cmd/sink/k8s/values.go

```diff
@@ -0,0 +1,18 @@
+// Copyright KubeArchive Authors
```

Comment:
Why does this merit its own package?

Reply:
This is not it's own package. This is one that already existed in the sink that provides the clients. Both  `cmd/sink/filters` and `cmd/sink/logs` need to determine what namespace this sink is in. I thought by putting it this package, which both can import, would reduces duplicated code.